make/golang.mk

```diff
@@ -52,9 +52,16 @@ golang-dep-vendor-deps: bin/dep
 # golang-godep-vendor is a target for saving dependencies with the dep tool
 # to the vendor/ directory. All nested vendor/ directories are deleted via
 # the prune command.
+# In CI, -vendor-only is used to avoid updating the lock file.
+ifndef CIRCLECI
```

Comment:
thoughts on using `CI` to be more generalized?

Reply:
This _is_ provided by circle ci

Reply:
👍 a0140b7